src/test/scenarios/managed-network/configuration/readme.az.md

```diff
@@ -29,11 +29,15 @@ python-sdk-output-folder: "$(output-folder)/src/managed-network/azext_managed_ne
 
 
 directive:
-    - where:
-        parameter-name: managed-network-group-name
-      set:
-        parameter-name: group-name
-#directive:
+  - where:
+      parameter-name: managed-network-group-name
+    set:
+      parameter-name: group-name
+  - from: swagger-document
+    where: $.definitions.ManagedNetwork.properties.properties
+    transform: >
+      $['x-ms-client-flatten'] = false;
+      
```

Comment:
why add this?

Reply:
to test manually set json that's the original purpose of this PR